### ISSUE TITLE
Has anyone successfully used Yolo to train custom data?

### ISSUE THREAD

**Issue by mshmoon** [ultralytics/yolov5#1794]
## ❔Question

I think the loss function of target detection task is really difficult to converge, I trained yolov3 before https://github.com/eriklindernoren/PyTorch-YOLOv3 This version of yolov3 has poor convergence. Now I'm training the current version of Yolov5, but I find that the convergence speed is very slow. Has anyone successfully trained custom data with yolo

**Comment by wudashuo** [comment]
Let me estimate conservatively, the number of people who successfully train custom data has exceeded 5000.
